docs/docs/target/git-repository.md

```diff
@@ -147,32 +164,60 @@ Total: 20 (UNKNOWN: 3, LOW: 0, MEDIUM: 7, HIGH: 5, CRITICAL: 5)
 
 </details>
 
-## Scanning a Branch
+### Misconfigurations
+It is disabled by default and can be enabled with `--scanners config`.
+See [here](../misconfiguration/scanning.md) for the detail.
+
+```shell
+$ trivy repo --scanners config [YOUR_REPO_URL]
+```
+
+### Secrets
+It is enabled by default.
+See [here](../secret/scanning.md) for the detail.
+
+```shell
+$ trivy repo [YOUR_REPO_URL]
+```
+
+### Secrets
```

Comment:
```suggestion
### Licenses
```

Reply:
Fixed in https://github.com/aquasecurity/trivy/pull/3503/commits/adc9fce4459cc882f49d138b56c5d6a805e45f31